src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubTriggerAttribute.cs

```diff
@@ -35,6 +36,7 @@ public EventHubTriggerAttribute(string eventHubName)
         /// <summary>
         /// Optional connection name. If missing, tries to use a registered event hub receiver.
         /// </summary>
+        [AppSetting]
         public string Connection { get; set; }
```

Comment:
I was looking for what used it. I found it in EventHubTriggerAttributeBinding.

---
In reply to: [113331328](https://github.com/Azure/azure-webjobs-sdk/pull/1119#discussion_r113331328) [](ancestors = 113331328)